reference/orders.v2.oas2.yml

```diff
@@ -1514,121 +1667,640 @@ components:
+                wrapping_cost_tax_class_id: 0
+                total_ex_tax: '931.86'
+                total_inc_tax: '1008.74'
+                total_tax: 76.88
+                is_tax_inclusive_pricing: false
+                items_total: 11
+                items_shipped: 0
+                payment_method: Test Payment Gateway
+                payment_provider_id: ''
+                payment_status: captured
+                refunded_amount: '0'
+                order_is_digital: false
+                store_credit_amount: '0'
+                gift_certificate_amount: '0'
+                ip_address: 70.112.53.67
+                geoip_country: United States
+                geoip_country_iso2: US
+                currency_id: 1
+                currency_code": USD
+                currency_exchange_rate: '1'
+                default_currency_id: 1
+                default_currency_code: USD
+                staff_notes: BIN-45
+                customer_message: Custom Journal Added
+                discount_amount: '0'
+                coupon_discount": 0
+                shipping_address_count: 1
+                ebay_order_id: '0'
+                cart_id: 8b84f622-faf1-4c10-887b-f5dff2f9eaf4
+                billing_address:
+                  first_name: Jane
+                  last_name: Doe
+                  company: ''
+                  street_1: 555 East Street
+                  street_2: ''
+                  city: Austin
+                  state: Texas
+                  zip: '78108'
+                  country: United States
+                  country_iso2: US
+                  phone: '1234567890'
+                  email: janedoe@email.com
+                  form_fields:
+                    - name: Delivery Instructions
+                      value: Leave in backyard
+                is_email_opt_in: false
+                credit_card_type: { }
+                order_source: manual
+                channel_id: 1
+                external_source: ''
+                products:
+                  url: 'https://api.bigcommerce.com/stores/{store_hash}/v2/orders/247/products'
+                  resource: /orders/247/products
+                shipping_addresses:
+                  url: 'https://api.bigcommerce.com/stores/{store_hash}/v2/orders/247/shippingaddresses'
+                  resource: /orders/247/shippingaddresses
+                coupons:
+                  url: 'https://api.bigcommerce.com/stores/{store_hash}/v2/orders/247/coupons'
+                  resource: /orders/247/coupons
+                external_id: null
+                external_merchant_id: null
+                tax_provider_id: BasicTaxProvider
+                store_default_currency_code: USD
+                store_default_to_transactional_exchange_rate: '1'
+                custom_status: Awaiting Fulfillment
+                customer_locale: en
+                external_order_id: external-order-id
+            Fee Added:
```

Comment:
Do we need these examples: "Fee added", "Fee updated", and "Fee deleted" ? They look quite the same, and they don't include the other fields on the "Order response" ? I'm not sure if it's helpful for client to have all 3 those examples.